pr47-core/src/vm/al31fu/cxx/include/aligned_alloc.h

```diff
@@ -0,0 +1,30 @@
+#ifndef PR47_AL31FU_ALIGNED_ALLOC_H
+#define PR47_AL31FU_ALIGNED_ALLOC_H
+
+#include <cstddef>
+#include <new>
+
+#include "imports.h"
+
+namespace pr47::al31fu {
+
+template <typename T>
+inline void* aligned_alloc_panic(size_t count) noexcept {
+#ifdef PR47_AL31FU_32BIT
+  // TODO: check overflow
+#endif
+
+  void *ret = static_cast<void*>(new (std::nothrow) size_t[count]);
+  if (!ret) {
+    pr47_al31fu_rs_rust_panic();
```

Comment:
According to [this](https://rust-lang.github.io/rfcs/2945-c-unwind-abi.html), unwinding across C/C++ frames can cause undefined behavior. I think we'd better use `abort`?